diff -Naur lcrypt/Makefile lcrypt.new/Makefile
--- lcrypt/Makefile	2010-12-16 16:21:35.000000000 +0000
+++ lcrypt.new/Makefile	2012-05-15 15:42:03.000000000 +0100
@@ -1,12 +1,18 @@
-CFLAGS += -O3 -Wall -DLITTLE_ENDIAN -DLTM_DESC -DLTC_SOURCE -DUSE_LTM -fPIC -I$(TOMCRYPT)/src/headers -I$(LUA)/src -I../src
+CFLAGS += -O3 -Wall -DLITTLE_ENDIAN -DLTM_DESC -DLTC_SOURCE -DUSE_LTM -fPIC \
+	-I$(TOMCRYPT)/src/headers -I$(LUA)/src -I../src
 
-$(PREFIX)/usr/local/lib/lua/5.1/lcrypt.so: lcrypt.o
-	mkdir -p $(PREFIX)/usr/local/lib/lua/5.1/
-	$(CC) $(CFLAGS) -shared -L$(TARGET)/lib -lm -lz -lutil -ltomcrypt -ltommath lcrypt.o -o $@
+all: lcrypt.so lcrypt.o 
+
+install: all
+	cp lcrypt.so $(TARGET)/lib/lua/5.1/
+
+lcrypt.so: lcrypt.o
+	$(CC) $(CFLAGS) -shared -L$(TARGET)/lib -lm -lz -lutil -ltomcrypt -ltommath -lluajit lcrypt.o -o $@
 
 lcrypt.o: lcrypt.c lcrypt_ciphers.c lcrypt_hashes.c lcrypt_math.c lcrypt_bits.c
 	$(CC) $(CFLAGS) -c lcrypt.c -o $@
-	$(STRIP) -g $@
 
 clean:
 	rm -f lcrypt.o lcrypt.so
+
+
diff -Naur lcrypt/lcrypt.c lcrypt.new/lcrypt.c
--- lcrypt/lcrypt.c	2010-12-31 22:55:03.000000000 +0000
+++ lcrypt.new/lcrypt.c	2012-05-15 15:49:41.000000000 +0100
@@ -19,7 +19,11 @@
   extern NFastApp_Connection nfast_conn;
   extern NFast_AppHandle nfast_app;
 #else
-  #include <pty.h>
+  #ifdef __APPLE__
+    #include <util.h>
+  #else
+    #include <pty.h>
+  #endif
 #endif
 
 #define likely(x)       __builtin_expect((x),1)
