diff -Naur libtommath/makefile libtommathnew/makefile
--- libtommath/makefile	2010-07-23 15:27:19.000000000 +0100
+++ libtommathnew/makefile	2012-05-15 13:34:23.000000000 +0100
@@ -52,9 +52,9 @@
 #LIBPATH-The directory for libtommath to be installed to.
 #INCPATH-The directory to install the header files for libtommath.
 #DATAPATH-The directory to install the pdf docs.
-DESTDIR=
-LIBPATH=/usr/lib
-INCPATH=/usr/include
+#DESTDIR=
+LIBPATH=/lib
+INCPATH=/include
 DATAPATH=/usr/share/doc/libtommath/pdf
 
 OBJECTS=bncore.o bn_mp_init.o bn_mp_clear.o bn_mp_exch.o bn_mp_grow.o bn_mp_shrink.o \
